snowpack/src/build/import-resolver.ts

```diff
@@ -99,10 +100,10 @@ export function createImportResolver({fileLoc, config}: {fileLoc: string; config
     if (config.packageOptions.external?.includes(spec)) {
       return spec;
     }
-    if (spec.startsWith('/')) {
+    if (spec[0] === '/') {
```

Comment:
Bikeshed: out of curiosity I benchmarked it, and this is faster than `.charAt()`!? I had always thought this was “slow” but seems to be optimized now